src/encoded/authentication.py

```diff
@@ -1,10 +1,817 @@
-from snovault.authentication import (
-    Auth0AuthenticationPolicy,
-    basic_auth_check,
-    BasicAuthAuthenticationPolicy,
-    CRYPT_CONTEXT,
-    generate_password,
-    generate_user,
-    NamespacedAuthenticationPolicy,
-    session_properties
+import base64
```

Comment:
What changes in this file are actually necessary vs. what is handled in snovault? I think most of this should be removed right?

Reply:
@willronchetti  this [commit](https://github.com/4dn-dcic/fourfront/pull/1864/commits/ec88bb87a125bea19376eac982c8dabc40eb63d3) was already in [redis_311](https://github.com/4dn-dcic/fourfront/tree/py311_redis). let me check it, if it became stale I'd remove.

Reply:
@willronchetti cleared the duplicate (and unnecessary) [stuff](https://github.com/4dn-dcic/fourfront/pull/1864/commits/592ee5693865170df77837b827fb6b6951f2ca2a) and redeployed it into mastertest/hotseat. They're still running well.